english/covid-19/0/0-CA.md

```diff
@@ -0,0 +1,13 @@
+You looked around and found a door that was not locked. Quickly, you broke the chain with all your power and ran to the door. Hearing the sound came the figures and they chased you. But, this was your day. You opened the door before the figures caught you and shut it.
+
```

Comment:
At least make one progressive link here which leads to somewhere else other than (../WIP.md)

Reply:
It would be much better if you remove the `[Add some options here](../WIP.md)` so that future contributors doesn't  feel limited to put only 3 links here (just a nit pick🙂)👍

Reply:
Thanks for your comments. But, I don't know how to progress the story. That's why I opened this PR. So, let any others complete it 😉 

> It would be much better if you remove the `[Add some options here](../WIP.md)` so that future contributors doesn't feel limited to put only 3 links here (just a nit pick🙂)👍

Ok. I will remove it.